consume-more-stuff/server/server.js

```diff
@@ -47,14 +49,32 @@ app.use(bodyParser.urlencoded({ extended: true }));
 //     });
 //   })
 
-app.use("/users", usersRouter);
-app.use("/items", itemsRouter);
-app.use("/categories", categoriesRouter);
+app.use("/api/users", usersRouter);
+app.use("/api/items", itemsRouter);
+app.use("/api/categories", categoriesRouter);
 
-app.get("/", (req, res) => {
-  console.log("smoke test");
-  res.send("smoke test");
-});
+app.post(`/api/login/:username`, (req, res) => {
+  let userUsername = req.params.username;
+  
+  return new User()
+  .where({ username: userUsername })
+  .fetch({
+    columns: ["username", "password"]
+  })
+  .then(data => {
+    if (!data) {
+      res.status(404).json({ message: `Username or password incorrect` })
```

Comment:
For a failed login the `401` status code is the proper code to return. `404` is page not found.